AjobK/Athena
# 340 login require direct

closes #340 

<br />

 **What is it supposed to do**  
When someone tries to use a functionality that requires the user to be logged in, a popup can be shown asking the user to log in. Doing so will redirect the user back to the page from whence those gents hath cometh.
![image](https://user-images.githubusercontent.com/14930277/144748097-966061ff-44a8-4609-b914-ad42a4335d1d.png)


<br />

 **How can we test your branch**  
Currently the only functionality that directly requires users to sign in is liking posts (commenting does as well, but this just shows a button for signing in instead). Try liking posts while logged in and logged out. See if you get redirected properly.

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers**

---
